Sources/MountebankSwift/Models/Stub/Response/Is.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+// swiftlint:disable type_name
+public struct Is: StubResponse, Equatable {
+    public static var defaultBehaviors: [Behavior] = []
+    public static var defaultHeaders: [String: String] = [:]
```

Comment:
These defaults can be set in a base test class to add default behavior and headers to all stubs.